src/foam/u2/detail/ChoiceSectionDetailView.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.detail',
+  name: 'ChoiceSectionDetailView',
+  extends: 'foam.u2.detail.AbstractSectionedDetailView',
+
+  documentation: `
+    A section-aware detail view that lets the user navigate between sections by
+    using arrow buttons to move forward and back, or a ChoiceView to navigate to
+    a specific section directly.
+  `,
+
+  requires: [
+    'foam.core.ArraySlot',
+    'foam.u2.detail.SectionView',
+    'foam.u2.view.ChoiceView'
+  ],
+
+  css: `
+    ^ .foam-u2-view-ScrollTableView table {
+      width: 100%;
+    }
+
+    ^ .foam-u2-tag-Select {
```

Comment:
This is probably going to affect all dropdowns that are inside of this detail view. We should add a specific class name to the ChoiceView for navigation and use that here instead of `.foam-u2-tag-Select`, which is too general